### PR TITLE
application/problem+json added as possible media type in successful responses

### PR DESCRIPTION
**Pull request by jesusjavierdediego**
Fixes #

## Proposed Changes

  - Inclusion of the application/problem+json in  200 OK responses as part of the async polling.

### CODE REVIEW

**Review by Amzani — APPROVED** [review]
LGTM

File: rest-api-guidelines/execution/long-running-tasks/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Long Running Tasks
+
+This section includes the recommended approaches to handling long runnint tasks (LRTs) in REST APIs.
```
Comment (Amzani) [inline rest-api-guidelines/execution/long-running-tasks/README.md:3]: @jesusjavierdediego  typo  `runnint`

**Review by Amzani — CHANGES_REQUESTED** [review]

File: rest-api-guidelines/execution/long-running-tasks/README.md (hunk 1 above)
Comment (jesusjavierdediego) [reply rest-api-guidelines/execution/long-running-tasks/README.md]: Fixed typos.
Changes in the cache section.

**Review by Amzani — APPROVED** [review]
LGTM

**Review by Amzani — APPROVED** [review]
LGTM

**Pull request merged**